src/Integration.UnitTests/Integration.UnitTests.csproj

```diff
@@ -42,8 +43,6 @@
 
     <PackageReference Include="Moq" Version="4.14.5" />
     <PackageReference Include="FluentAssertions" Version="5.9.0" />
-    <PackageReference Include="System.IO.Abstractions" Version="9.0.4" />
-    <PackageReference Include="System.IO.Abstractions.TestingHelpers" Version="9.0.4" />
```

Comment:
Two of the test projects were also referencing `System.IO.Abstractions.TestingHelpers`, but I removed the references since they didn't seem to be using it.